Sources/MapboxNavigation/NightStyle.swift

```diff
@@ -108,5 +108,11 @@ open class NightStyle: DayStyle {
         TimeRemainingLabel.appearance().trafficUnknownColor = #colorLiteral(red: 0.9842069745, green: 0.9843751788, blue: 0.9841964841, alpha: 1)
         TopBannerView.appearance().backgroundColor = backgroundColor
         WayNameView.appearance().borderColor = #colorLiteral(red: 0.2802129388, green: 0.3988235593, blue: 0.5260632038, alpha: 1)
+        
+        UILabel.appearance(whenContainedInInstancesOf: [FeedbackViewController.self]).backgroundColor = .black
```

Comment:
Can you also handle case when style changes during active navigation. For example:

![Screen Shot 2022-03-03 at 12 55 05 PM](https://user-images.githubusercontent.com/1496498/156651495-17d4f602-ed7f-48ff-b50e-dac649bd1c79.png)

Reply:
When the `FeedbackViewController` is already presented and its view entered the window, the style change and the `UIAppearance` update won't automatically update its view appearance. So we need to remove it from the super view and then add it back. 

Currently it's fixed during active navigation. And the example to update the style in passive navigation is also added.